deepchem/data/datasets.py

```diff
@@ -2713,7 +2713,8 @@ def get_shape(self) -> Tuple[Shape, Shape, Shape, Shape]:
 
     def get_label_means(self) -> pd.DataFrame:
         """Return pandas series of label means."""
-        return self.metadata_df["y_means"]
+        label_means = np.mean(self.metadata_df['y'],axis=0)
```

Comment:
I don't think this will work correctly; there is a reason why we compute the y_means as specified here. Read through the full class and look at the existing tests. I think something is going to break but I will have to review more carefully

Reply:
I see in this "metadata_df"  among doesn't have a 'y_means' column. So I try to simply calculate label means into this function.

Reply:
If anyone sees my changes?

Reply:
Does anyone see my changes? If not then I remove this repository becuse it takes a lot of time to get answers.